masterlist.yaml

```diff
@@ -4916,6 +4916,19 @@ plugins:
         itm: 1
         udr: 0
         nav: 0
+  - name: 'RobCo Certified.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/43331' ]
+    tag:
+      - Factions
+    dirty:
+      - <<: *quickClean
+        crc: 0x3B28B2FE
+        util: '[FNVEdit v4.0.1i EXPERIMENTAL](https://www.nexusmods.com/newvegas/mods/34703)'
+        itm: 9
+    clean:
```

Comment:
It looks like this shouldn't be there.

Reply:
Yes. I had originally added load order requirements but they are alreaday added as masters, and as such automatically detected. I will however make one update, there should be NVSE requirement listed.

Also, sorry I'm new to this github stuff lol

Reply:
A plugin can't be clean and also have wild edits... The wild edit list item should also come under `dirty:`.

Reply:
FTFY

Reply:
Same as for #57, the fix hasn't appeared here.

Reply:
Same as #55, appears on my end.

https://github.com/gutfuckllc/falloutnv/commit/021fba321ccbf79238c1fbfba04509423283360e